Add template-glib

astavale opened the pull request:
Fixes #151

benwaffle commented:
is there no Template-1.0.deps file? If it exists, you don't need to specify `--pkg gio-2.0`

astavale commented:
The VAPI in https://github.com/nemequ/vala-girs/tree/master/vala/vapi is called `template-glib-1.0.vapi` and does have a deps file. What I did though was download Template-1.0.gir and built the docs from that. `valadoc` required the `--pkg gio-2.0`. So I added it to `packages.xml`. I've removed that now.
